### ISSUE TITLE
Need more colours

### ISSUE THREAD

**Issue by StuartLittlefair** [HiPERCAM/hipercam#8]
and names

**Comment by StuartLittlefair** [comment]
Note that any colour in the [xkcd colour survey](https://blog.xkcd.com/2010/05/03/color-survey-results/) can be derived by name in matplotlib, i.e

plt.plot(x, y, color='xkcd:sky blue')

**Comment by StuartLittlefair** [comment]
and we can get rgb tuples

```python
from matplotlib import colors
colors.to_rgb('xkcd:sky blue')
```

**Comment by StuartLittlefair** [comment]
Look [here](https://xkcd.com/color/rgb/) and see what you like

**Comment by trmrsh** [comment]
Quite a few colours have now been added.  They can be seen in hipercam/core.py ['CNAMS'], and if you type an unrecognised colour in a reduce file, reduce will give you a warning and list the recognised colours. I think we have settled for now on purple, green, orange, red, ]darkred or mud] to represent u,g,r,i,z. If you want to repeat the colours in your own code, hipercam.rgb returns the rgb tuple given a string name.
